recipe/meta.yaml

```diff
@@ -131,4 +131,3 @@ extra:
     - jakirkham
     - ocefpaf
     - mingwandroid
-    - xylar
```

Comment:
I enjoy helping out on feedstock where I feel I have enough background to contribute.  This doesn't happen to be one of those.